.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+## TL;DR
+
+<!-- Put 1-2 sentences here, at max. -->
+
+## Depends on
+
+<!-- add a link here for something this PR depends on, like another PR -->
+
+## Issues
+
+<!-- Paste any issue links here, even issues from Gitlab --> 
+
+## Changelog
+
+- Upgraded dependency from 1.0 to 1.1
+- Ripped out `oldFunction()`
+- Combined `file1` and `file2`
+
+## Follow up items
+
+- [ ] Update this other repo
+- [ ] Make a post in `#demo`
+- [ ] Failover some service
+
+## Tophatting
```

Comment:
I like the Tophatting examples you've done in other PRs! However, before I saw you do this I had no idea what "tophatting" was... I think it would be helpful to briefly define "tophatting" or even rename it to something equivalent

Reply:
Same - Maybe we can use an "comment" to describe it so it doesn't show up in the rendered document but is there when someone goes to write.  The monolith devs do this.